### DISCUSSION TITLE
Support for UWP apps on the Windows 10 Start Menu in Windows 11

### DISCUSSION THREAD

**Discussion by B1ue-Dev** [valinet/ExplorerPatcher#1227] [Ideas]
I do use Microsoft Store apps, as some of them are pretty neat, the thing is, setting the Start Menu to the Windows 10 one will only show Win32 apps, not UWP. Is this a bug or not planned yet?
If it is not planned yet, I hope this will be a thing in the next update.

**Comment by B1ue-Dev** [comment]
Image in case anyone wonder what I am saying.
![image](https://user-images.githubusercontent.com/60958064/229179306-72ccf58e-34c4-4989-a0ad-5a7a1cc3a605.png)

**Reply by topia** [reply]
See #1108. Although it's a bug, there is no exact plan for a fix now.
